_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,441 @@
+			-ms-transform: rotate(20deg); /* IE 9 */
+			-webkit-transform: rotate(20deg); /* Safari 3-8 */
+			transform: rotate(20deg);
+		}
+	</style>
+</head>
+<body>
+	<div class="tilt">look at this pieCe of tiLted text fOr clueS on whEre to find The monster.</div>
+</body>
+```
+
+#### Passed Example 5
+
+This paragraph includes the [visual reference word][] "round". The button is identified by this word which is also included in the [visible text content][] of the element, thus matching the (**visible words**) condition.
+
+```html
+<head>
+	<title>Passed example 5 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click the round button, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Round button</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 6
+
+This paragraph includes the [visual reference word][] "triangle" which is included in the heading of the menu, thus matching the (**visible words**) condition. The fact that the described content is on another page of the same website does not restrict this rule.
+
+```html
+<p>
+	On the <a href="/test-assets/SC1.3.3-triangle-menu-with-heading.html">information page</a> you can find more examples
+	within the triangle menu.
+</p>
+```
+
+#### Passed Example 7
+
+This paragraph includes the [visual reference words][] "wide" and "narrow". The images are indicated by these words which are also included in the [accessible names][accessible name] of the images, thus matching the (**accessible words**) condition.
+
+```html
+<p>Look at the wide image which is awesome. Now compare it with the narrow image which isn't.</p>
+<img scr="/test-assets/images/awesome_wide.jfif" alt="Wide photo of an awesome landscape." />
+<img scr="/test-assets/images/Non_awesome_narrow.jpg" alt="Narrow photo of a dull landscape." />
+```
+
+#### Passed Example 8
+
+This paragraph includes the [visual reference word][] "square" but in this case it is not identifying any [web content][], thus matching the (**no indication**) condition.
+
+```html
+<p>A square is a regular quadrilateral with four equal sides and four right angles.</p>
+```
+
+#### Passed Example 9
+
+This paragraph includes the [visual reference word][] "circle" but in this case it is not an instruction about any [web content][], thus matching the (**no instruction**) condition.
+
+```html
+<p>
+	This circle is nice.
+	<span
+		style="height: 25px;
+		     width: 25px;
+		     background-color: #bbb;
+		     border-radius: 50%;
+		     display: inline-block;"
+	>
+	</span>
+</p>
+```
+
+#### Passed Example 10
+
+This paragraph includes the [visual reference word][] "star" but there is also a heading "examples" that can be referenced, thus matching the (**non-visual references**) condition. The fact that the identified content is in an `iframe` does not restrict this rule.
```

Comment:
There is only "*" in <code>test-assets/SC1.3.3-star-with-heading.html</code> and <code>test-assets/SC1.3.3-star-without-heading.html</code> which is not a visual reference word "star". Should be updated accordingly. Otherwise this example will be considered as failed.